adhocracy-plus/assets/js/app.js

```diff
@@ -53,6 +54,10 @@ function init () {
     slidesToShow: 1,
     slidesToScroll: 1
   })
+
+  if ($.fn.select2) {
+    $('.js-select2').select2()
+  }
 }
```

Comment:
I'm leaving this in until we know which we'll use but I'm not calling it on the select in this pull request

Reply:
:+1: